scripts/autotools/libmongoc/CheckSSL.m4

```diff
@@ -188,18 +188,24 @@ if test "$PHP_MONGODB_SSL" = "openssl" -o "$PHP_MONGODB_SSL" = "libressl" -o "$P
     AC_SUBST(MONGOC_ENABLE_SSL_SECURE_TRANSPORT, 1)
     AC_SUBST(MONGOC_ENABLE_CRYPTO_LIBCRYPTO, 0)
     AC_SUBST(MONGOC_ENABLE_CRYPTO_COMMON_CRYPTO, 1)
+
+    PHP_MONGODB_BUNDLED_CFLAGS="$PHP_MONGODB_BUNDLED_CFLAGS -DKMS_MESSAGE_ENABLE_CRYPTO=1 -DKMS_MESSAGE_ENABLE_CRYPTO_COMMON_CRYPTO=1"
   elif test "$PHP_MONGODB_SSL" = "openssl"; then
     AC_SUBST(MONGOC_ENABLE_SSL_OPENSSL, 1)
     AC_SUBST(MONGOC_ENABLE_SSL_LIBRESSL, 0)
     AC_SUBST(MONGOC_ENABLE_SSL_SECURE_TRANSPORT, 0)
     AC_SUBST(MONGOC_ENABLE_CRYPTO_LIBCRYPTO, 1)
     AC_SUBST(MONGOC_ENABLE_CRYPTO_COMMON_CRYPTO, 0)
+
+    PHP_MONGODB_BUNDLED_CFLAGS="$PHP_MONGODB_BUNDLED_CFLAGS -DKMS_MESSAGE_ENABLE_CRYPTO=1 -DKMS_MESSAGE_ENABLE_CRYPTO_LIBCRYPTO=1"
   elif test "$PHP_MONGODB_SSL" = "libressl"; then
     AC_SUBST(MONGOC_ENABLE_SSL_OPENSSL, 0)
     AC_SUBST(MONGOC_ENABLE_SSL_LIBRESSL, 1)
     AC_SUBST(MONGOC_ENABLE_SSL_SECURE_TRANSPORT, 0)
     AC_SUBST(MONGOC_ENABLE_CRYPTO_LIBCRYPTO, 1)
     AC_SUBST(MONGOC_ENABLE_CRYPTO_COMMON_CRYPTO, 0)
+
+    PHP_MONGODB_BUNDLED_CFLAGS="$PHP_MONGODB_BUNDLED_CFLAGS -DKMS_MESSAGE_ENABLE_CRYPTO=1 -DKMS_MESSAGE_ENABLE_CRYPTO_LIBCRYPTO=1"
```

Comment:
Just to confirm, this will get incorporated into `PHP_MONGODB_BUNDLED_CFLAGS` _before_  `PHP_MONGODB_LIBMONGOCRYPT_CFLAGS` pulls in `PHP_MONGODB_BUNDLED_CFLAGS` (see: https://github.com/alcaeus/mongo-php-driver/blob/4df86904ea471405fd27ed50befa452bfc174671/config.m4#L436), correct?

Reply:
Yes, correct. Those flags were only set in `PHP_MONGODB_LIBMONGOCRYPT_CFLAGS` but will now always be set if we find an SSL library.